labs/iot/gps-lab/server/code/static/js/application.js

```diff
@@ -0,0 +1,60 @@
+
+var map, datasource, client, popup, searchInput, resultsPanel, searchInputLength, centerMapOnResults;
+
+//The minimum number of characters needed in the search input before a search is performed.
+var minSearchInputLength = 3;
+
+//The number of ms between key strokes to wait before performing a search.
+var keyStrokeDelay = 150;
+
+function GetMap()
+{
+    //Initialize a map instance.
+    map = new atlas.Map('myMap', {
+        center: [-122.64, 47.58],
+        zoom: 4,
+        view: "Auto",
+        style: "satellite",
+        authOptions: {
+            authType: 'subscriptionKey',
+            subscriptionKey: '<Azure_Map_Subscription_Key>'
```

Comment:
Can this be factored out so it's easier to update, or passed in from the Python code?